### ISSUE TITLE
Nothing announced when following links targeting text inputs or textareas

### ISSUE THREAD

**Issue by 36degrees** [alphagov/reported-bugs#5]
**Upstream bug:** https://bugs.webkit.org/show_bug.cgi?id=191746
**Date:** 2018-11-16
**Reported by:** Oliver Byford
**Related to:** https://github.com/alphagov/govuk-frontend/pull/1056#issuecomment-439381905

## Overview

Given the following markup:

```
  <a href="#textinput">Link to input type="text"</a><br>
  <a href="#textarea">Link to textarea</a>

  <br><br>

  <label for="textinput">Label for text input</label><br>
  <span id="textinput-description">Description for text input</span><br>
  <input id="textinput" type="text" aria-describedby="textinput-description" value="Foo">

  <br><br>

  <label for="textarea">Label for textarea</label><br>
  <span id="textarea-description">Description for textarea</span><br>
  <textarea id="textarea" name="textarea" rows="10" cols="50" aria-describedby="textarea-description">Bar</textarea>
```

Focus the link to each input type and activate it by double-tapping.

### Expected behaviour:

Voiceover should announce the same context that it announces when focusing the field by swiping through the document, including the label, field type, current value, and any description:

"Label for text input, text field, foo. Description for text input. Double-tap to edit"

### Actual behaviour:

The VO cursor moves to the target input, but nothing is announced.

### Expected behaviour

Voiceover should announce the same context that it announces when focusing the field by swiping through the document, including the label, field type, current value, and any description:

"Label for textarea, multi-line text field, bar. Description for textarea. Double-tap to edit"

### Actual behaviour:

The VO cursor moves to the target input, but nothing is announced.

Tested in iOS 12.1 (16B92) on an iPhone X (A1901)

**Comment by querkmachine** [comment]
Re-tested in iOS 26.4 and this issue is still present.
